### DISCUSSION TITLE
Configuration DNSBL

### DISCUSSION THREAD

**Discussion by davaxi** [postalserver/postal#1474] [Feature suggestions]
Hi,

How it's possible to filter SPAM email with DNSBL check ? In postfix exist configuration : reject_rbl_client but in postal, not existing.

Thanks

**Comment by willpower232** [comment]
At the minute this isn't part of Postal but it would be a good feature request if you get lots of incoming email to your Postal server.

**Comment by influenist** [comment]
Is there an update on this matter?

**Comment by willpower232** [comment]
Unless someone has a PR in the works, I don't believe so
